apps/potterhat_node/lib/node/node.ex

```diff
@@ -18,7 +18,7 @@ defmodule PotterhatNode.Node do
   """
   use GenServer
   require Logger
-  alias PotterhatNode.EventLogger
+  alias PotterhatNode.{ActiveNodes, EventLogger}
```

Comment:
I don't like these because it's harder to refactor. Whats your experience?

Reply:
Not a big issue for me 😂

Reply:
Team #AliasWasAMistake 😜

Reply:
Actually I agree with #AliasWasAMistake, especially with credo enforcing aliases at all time. 

There are some cases where I find aliases useful (e.g. deep namespaces) and many other times where aliases make thing more confusing (modules with similar names when stripped of namespaces).

And resorting to alias's `:as` option makes thing even worse because now the module reference may not always have the same name. Having longer, more descriptive module names also works but they can be really eye-cluttering too.

Let's talk more about aliases out of this PR? 😂